ibm/service/satellite/resource_ibm_satellite_location.go

```diff
@@ -93,6 +94,18 @@ func ResourceIBMSatelliteLocation() *schema.Resource {
 				},
 				Description: "The IBM Cloud metro from which the Satellite location is managed",
 			},
+			"physical_address": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "An optional physical address of the new Satellite location which is deployed on premise",
+			},
+			"capabilities": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Set:         schema.HashString,
+				Description: "The satellite capabilities attached to the location",
+			},
```

Comment:
I'm not familiar with the API, but in case these can be only set once during create time and not updated we need `DiffSuppressFunc: flex.ApplyOnce`

Reply:
Yes, you are right that physical_address and capabilites cannot be changed after creation.

Thanks. Done